src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -384,6 +384,24 @@ void InitializeCurrentProcessCpuCount()
     g_RhNumberOfProcessors = count;
 }
 
+static uint32_t g_RhPageSize;
+
+void InitializeOsPageSize()
+{
+    g_RhPageSize = (uint32_t)sysconf(_SC_PAGE_SIZE);
```

Comment:
Perhaps better:
```c
#ifdef OS_PAGE_SIZE
    g_RhPageSize = (uint32_t)OS_PAGE_SIZE;
#else
    g_RhPageSize = (uint32_t)sysconf(_SC_PAGE_SIZE);
#endif
```

, delete `PalOsPageSize` and use `PalGetOsPageSize` everywhere?

nit: Os -> OS (two-letter abbreviation rule)

Reply:
> delete PalOsPageSize and use PalGetOsPageSize everywhere?

I think we need one that inlines, and one that does not.
I did not replace uses of `OS_PAGE_SIZE`, though, as that would be a lot of changes and keeping `OS_PAGE_SIZE` would match closer the similar pattern in GC code.